osc-ui/src/main/java/org/osc/core/broker/view/maintenance/NetworkLayout.java

```diff
@@ -87,28 +79,32 @@ public NetworkLayout(GetNetworkSettingsServiceApi getNetworkSettingsService,
         super();
         this.getNetworkSettingsService = getNetworkSettingsService;
         this.checkNetworkSettingsService = checkNetworkSettingsService;
-        this.setNetworkSettingsService = setNetworkSettingsService;
         this.getNATSettingsService = getNATSettingsService;
         this.setNATSettingsService = setNATSettingsService;
         this.server = server;
+        this.validator=validator;
```

Comment:
where is this used?

Reply:
Its used in MaintenanceView buildNetworkForm() while reading or writing the NAT value.